nanoservice/client.py

```diff
@@ -42,14 +44,18 @@ def __init__(self, addr, encoder=None, socket=None,
         self.sock.connect(self.addr)
         self.authenticator = Authenticator(secret, digestmod) if auth else None
 
-    def build_payload(self, method, args):
+    @classmethod
```

Comment:
Why make this `classmethod`? (Is it because of pylint complaining?)

I'm inclined towards making this an instance method, since building a payload might later become dependent on the client type instantiated. I think it's an simpler mental model, knowing that building a payload is tied the client instance, rather than client class.

Contrived Example:

```python
client = Client(..., someFutureFlag=True)

def build_paylod(self, method, args):
    if self.someFutureFlag:
        return special_build_payload(method, args)
    return normal_build_payload(method, args)
```